packages/vite/types/importMeta.d.ts

```diff
@@ -2,8 +2,10 @@
 // Thus cannot contain any top-level imports
 // <https://www.typescriptlang.org/docs/handbook/declaration-merging.html#module-augmentation>
 
+type ImportMetaKey = ViteStrictImportMetaEnv extends true ? never : string
```

Comment:
```suggestion
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore ViteStrictImportMetaEnv may or may not be declared by users
type ImportMetaKey = ViteStrictImportMetaEnv extends true ? never : string
```
Ah, you can add a lint ignore comment like https://github.com/vitejs/vite/blob/f7b1964d3a93a21f80b61638fa6ae9606d0a6f4f/packages/vite/types/internal/cssPreprocessorOptions.d.ts#L1

Reply:
Thank you so much ❤️